recipes/esmvaltool/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+xorg-x11-server-Xorg
```

Comment:
Why do we need x11? I'm pretty sure our users are running the tool on systems without a graphical user interface.

Reply:
Removing x11 dependencies is exactly why we need this. It came in as part of conda-forge/esmvaltool-python-feedstock#1 when we moved from depending on `matplotlib` to `matplotlib-base` which removes the `pyqt` dependency and a whole bunch of x11 dependencies. To still be able to build, we need this yum requirement on the build machine.